product_docs/docs/pgd/5/rel_notes/pgd_5.5.0_rel_notes.mdx

```diff
@@ -0,0 +1,86 @@
+| BDR       | 5.5.0   | Grant additional object permissions to role "bdr_read_all_stats"                                                              |                |
+| BDR       | 5.5.0   | Improve stability of manager worker and Raft consensus by not throwing error on non-fatal dynamic shared memory read failures |                |
+| BDR       | 5.5.0   | Improve stability of Raft consensus and workers by handling dynamic shared memory errors in the right place                   |                |
+| BDR       | 5.5.0   | Expose number of changes processed by writer in a large transaction                                                           |                |
+| BDR       | 5.5.0   | Expose heatbeat timings in Raft get_raft_status()                                                                             |                |
+| BDR       | 5.5.0   | Expose follower info in Raft get_raft_status() on Raft leader nodes                                                           |                |
+| BDR       | 5.5.0   | Expose number of changes processed by writer in a large transaction                                                           |                |
+| BDR       | 5.5.0   | Stop the initial replication connection for bdr_init_physical and start it only when needed                                   | RT102828/35305 |
+| BDR       | 5.5.0   | Grant usage of `pg_file_settings` and `pg_show_all_file_settings()` to bdr_superuser                                          |                |
+| CLI       | 5.5.0   | Add new read scalability related options to JSON output of `show-proxies ` and `show-groups` commands                         |                |
+| CLI       | 5.5.0   | Add new option called `proxy-mode` to `create-proxy` command for read scalability support                                     |                |
+| CLI       | 5.5.0   | Add raft leader in  tabular output of `show-groups` command                                                                   |                |
+
+
+## Bug fixes
+
+| Component | Version | Description                                                                                                               | Addresses      |
+|-----------|---------|---------------------------------------------------------------------------------------------------------------------------|----------------|
+| BDR       | 5.5.0   | Improve handling of node group configuration parameter "check_constraints"                                                | RT99956/31896  |
+| BDR       | 5.5.0   | Fix incorrect parsing of pre-commit message that caused nodes to diverge on commit decision for group commit transactions |                |
+| BDR       | 5.5.0   | Prevent potential segfault in bdr.monitor_group_versions()                                                                | RT102290/34051 |
```

Comment:
```suggestion
| BDR       | 5.5.0   | Fixed an issue to prevent potential segfault in `bdr.monitor_group_versions()`.                                                                | RT102290/34051 |
```